### ISSUE TITLE
Parse of REMARK into Metar

### ISSUE THREAD

**Issue by umpire274** [mivek/python-metar-taf-parser#2]
Is it in your plan to insert the parsing of 'REMARK' section into METAR string?

**Comment by mivek** [comment]
Hello @umpire274 

The remark part of the metar is stored in the `remark` property of the[ `metar` object.](https://github.com/mivek/python-metar-taf-parser/blob/main/metar_taf_parser/model/model.py#L358)

It is not parsed like the other parts of the metar because of the many different information it contains. The idea would be to decrypt the REMARK part and translate it in plain english, like it is done in the [java version](https://github.com/mivek/MetarParser) of the project.

**Comment by umpire274** [comment]
OK thank you.

> ... The idea would be to decrypt the REMARK part and translate it in plain english, like it is done in the java version

Need help for this goal?

**Comment by mivek** [comment]
@umpire274 
Update on the topic: I am currently working on the parsing of the remarks.
I will create a PR in the next days.
